README-ZH.md

```diff
@@ -111,7 +111,7 @@ UIWidgets应用是用**C＃脚本**来编写的。 请按照以下步骤创建
 
     namespace UIWidgetsSample
     {
-        public class CountDemo : UIWidgetsPanel
+        public class UIWidgetsExample : UIWidgetsPanel
```

Comment:
The class name has to match the script file name in order for the script to be attached to the canvas panel.